### PR TITLE
lazy one

### PR DESCRIPTION
**Pull request by iamsabin**

### CODE REVIEW

**Comment by morosebose** [comment]
@casey-collab

File: nepali/aalchi lagyo/alchi.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Yo k k ho description ma padnai alchi lagyo
```
Comment (casey-collab) [inline nepali/aalchi lagyo/alchi.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by iamsabin** [comment]
I have now fixed the link, review @casey-collab

File: nepali/aalchi lagyo/alchi.md (hunk 1 above)
Comment (casey-collab) [inline nepali/aalchi lagyo/alchi.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request merged**